packages/preview/sdu-typst-touying/0.1.0/lib.typ

```diff
@@ -0,0 +1,512 @@
+#import "dependency.typ": *
+#let sdu_red=rgb("#880000")
```

Comment:
Typst packages generally use kebab-case names (and you seem to follow that convention in the rest of your package). Can you please rename this variable to `sdu-red`?